<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Browser</class>
 <widget class="QWidget" name="Browser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter_2">
     <property name="styleSheet">
      <string notr="true">QSplitter::handle {    
    background-color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="splitter">
      <property name="styleSheet">
       <string notr="true">QSplitter::handle {    
    background-color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTextBrowser" name="textBrowser"/>
      <widget class="QPlainTextEdit" name="plainTextEdit"/>
     </widget>
     <widget class="QWidget" name="">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButtonForeward">
         <property name="text">
          <string>前进</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonBack">
         <property name="text">
          <string>后退</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonOpen">
         <property name="text">
          <string>打开html</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
